submissions/andrewklmn/html-css-cinema-booking/index.html

```diff
@@ -44,9 +44,9 @@ <h2 class="hidden">Crew</h2>
           <p><strong>Cast:</strong> Matthew McConaughey, Anne Hathaway, Jessica Chastain and others</p>
         </section> 
       </section>
-      <section class="interactive" aria-label="Area for setting of booking details">
+      <section class="form_area" aria-label="Area for setting of booking details">
         <h2 class="hidden">Ticket booking form</h2>      
-        <form class="booking_form" action="index.html" method="POST" autocomplete="off" aria-label="Interactive area">
+        <form class="booking_form" action="index.html" method="POST" autocomplete="off" aria-label="form_area area">
```

Comment:
Value of `aria-label` will be spoken by screenreader and text must provide the label of the current block.